data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+	source "Ki Patek Ka"
+	destination "Inmost Blue"
+	to offer
+		has "Fangvin: Struggling Author 3: done"
+	on offer
+		conversation
+			`The spaceport of Ki Patek Ka is vast. You decide to search for Anna, who has chosen to settle in the cafeteria, with her laptop, She is typing away at the keyboard, presumably writing a new novel.`
+			`	"<first>! I found a publishing company who's willing to publish my book! I sent them a digital copy of my book and they have instructed me to visit their office on <destination>!", exclaims Anna, once again in an enthusiastic tone.`
+			choice
+				`	"What's the publishing company called?"`
+			`	"They're called 'Kimek Reader Publishing'.`
+			choice
+				`	"Great!"`
+				`	"Let's go!"`
+			`	Anna closes her laptop, places her things in her bag, and begins walking to your ship. You and Anna enter the ship, as she settles into her bunk. It's time to go to <destination>, where the main offices of 'Kimek Reader Publishing' are located.`
+				accept
+	on complete
+		event "beyond the hyperdrive is published" 165 197
+		conversation
+			`Landing on <planet>, Anna is excitedly holding a copy of 'Beyond the Hyperdrive', her novel about humans stealing jump drives from the Quarg. She hastily opens the hatch as soon as it unlocks.`
+			choice
+				`	"Wait for me!"`
+			`	Anna hears your scream and stops. "We're here, Anna. Now, we need to find their offices.", you say.`
+			`	"That won't be necessary. In the message they sent me, they included precise instructions on how to reach their offices."`
+			`	You begin following Anna to the publisher's offices. You walk through endless rows of shops and streets, take a train to a large white building, and eventually find what looks like the Kimek equivalent of an elevator.`
+			`	You and Anna enter the elevator, as Anna presses a button that closes the door and takes you up. When the door opens, you see a Kimek in front of you, a necklace around it with ornate jewelry and a translation device.`
+			`	"Greetings, humans. Follow me, you must, so publish your book, we can.", says the Kimek through its translation box.`
+			`	At the direction of the Kimek, you and Anna begin walking until directed to a room with glass walls. Anna puts down the copy of her book on the table, and the three of you take a seat.`
+			choice
+				`	"Hello. My name is <first> <last>, and this is my friend Anna Fangvin."`
+			`	"Greetings. At Kimek Reader Publishing, I work. A human, I have never encountered. But the book you sent, translate it we were able to."`
+			`	"Read it, we still haven't, and run it past the Heliarch censors, we must."`
+			`	Two Heliarch agents enter the room. "Supervise the interview, we are obligated to."`
+			choice
+				`	"How long will I have to wait?"`
+					goto "how long"
+				`	"Should I stay here to accompany Anna?`
+					goto "should I accompany anna"
+			label "how long"
+			`	"Wait a few hours, you must. But quick, this will be.", says the Kimek, interviewing you.`
```

Comment:
It's either the case that it'll take hours, or that it'll be quick. Having both things said here makes it very weird to read. Since it is a lengthy process, just have it so the player is told the process will take hours.

Reply:
```suggestion
			`	"Wait a few hours, you must," explains the Kimek.`
```